src/os/windows/pws_autotype/CMakeLists.txt

```diff
@@ -1,7 +1,45 @@
+# Version configuration:
+# versionDLLs.mfc for Windows MFC build
+# version file has the following format:
+# AT_VER_MAJOR = n
+# AT_VER_MINOR = n
+# AT_VER_REV = n (default 0)
+# where VER_MAJOR and VER_MINOR are mandatory, and all must be integers
+
+set (VERSION_FILE "${PROJECT_SOURCE_DIR}/versionDLLs.mfc")
```

Comment:
I think it would be more elegant to fold these (and the other DLL versions) into the versions.mfc file.